examples/influxdb_data_model.ipynb

```diff
@@ -0,0 +1,511 @@
+      "Sending InfluxDB line for job 3256.\n",
+      "Sending InfluxDB line for job 3257.\n",
+      "Sending InfluxDB line for job 3258.\n",
+      "Sending InfluxDB line for job 3259.\n",
+      "Sending InfluxDB line for job 3260.\n",
+      "Sending InfluxDB line for job 3261.\n",
+      "Sending InfluxDB line for job 3262.\n",
+      "Sending InfluxDB line for job 3263.\n",
+      "Sending InfluxDB line for job 3264.\n",
+      "Sending InfluxDB line for job 3265.\n",
+      "Sending InfluxDB line for job 3266.\n",
+      "Sending InfluxDB line for job 3267.\n",
+      "Sending InfluxDB line for job 3268.\n",
+      "Sending InfluxDB line for job 3269.\n",
+      "Sending InfluxDB line for job 3270.\n",
+      "Sending InfluxDB line for job 3271.\n",
+      "Sending InfluxDB line for job 3272.\n",
+      "Sending InfluxDB line for job 3273.\n",
+      "Sending InfluxDB line for job 3274.\n",
+      "Sending InfluxDB line for job 3275.\n",
+      "Sending InfluxDB line for job 3276.\n",
+      "Sending InfluxDB line for job 3277.\n",
+      "Sending InfluxDB line for job 3278.\n",
+      "Sending InfluxDB line for job 3279.\n",
+      "Sending InfluxDB line for job 3280.\n",
+      "Sending InfluxDB line for job 3281.\n",
+      "Sending InfluxDB line for job 3282.\n",
+      "Sending InfluxDB line for job 3283.\n",
+      "Sending InfluxDB line for job 3284.\n",
+      "Sending InfluxDB line for job 3285.\n",
+      "Sending InfluxDB line for job 3286.\n",
+      "Sending InfluxDB line for job 3287.\n",
+      "Sending InfluxDB line for job 3288.\n",
+      "Sending InfluxDB line for job 3289.\n",
+      "Sending InfluxDB line for job 3290.\n",
+      "Sending InfluxDB line for job 3291.\n",
+      "Sending InfluxDB line for job 3292.\n",
+      "Sending InfluxDB line for job 3293.\n",
+      "Sending InfluxDB line for job 3294.\n",
+      "Sending InfluxDB line for job 3295.\n",
+      "Sending InfluxDB line for job 3296.\n",
+      "Sending InfluxDB line for job 3297.\n",
+      "Sending InfluxDB line for job 3298.\n",
+      "Sending InfluxDB line for job 3299.\n",
+      "Sending InfluxDB line for job 3300.\n",
+      "Sending InfluxDB line for job 3301.\n",
+      "Sending InfluxDB line for job 3302.\n",
+      "Sending InfluxDB line for job 3303.\n",
+      "Sending InfluxDB line for job 3304.\n",
+      "Sending InfluxDB line for job 3305.\n",
+      "Sending InfluxDB line for job 3306.\n",
+      "Sending InfluxDB line for job 3307.\n",
+      "Sending InfluxDB line for job 3308.\n",
+      "Sending InfluxDB line for job 3309.\n"
+     ]
+    }
+   ],
+   "source": [
+    "jobs = requests.get(SQUASH_API_URL + \"/jobs\").json()\n",
+    "\n",
+    "for job_id in jobs['ids']:\n",
+    "    \n",
+    "    data = requests.get(SQUASH_API_URL + \"/job/{}\".format(job_id)).json()\n",
+    "    \n",
+    "    # Skip deprecated datasets\n",
+    "    if data['ci_dataset'] == 'unknown' or data['ci_dataset'] == 'decam':\n",
```

Comment:
They've been telling us we can turn on decam in CI again.  I don't know if that changes how you want to handle things here.

Reply:
I would just remove this restriction once `decam` is turned on in CI again.